test/Utils.ts

```diff
@@ -5,6 +5,9 @@ import path from 'path';
 import { DeploymentResult } from './TestSetup';
 import { RelayHubConfiguration } from '@rsksmart/rif-relay-common';
 
+export const RIF_RELAY_HOST = 'http://localhost';
+export const RIF_RELAY_PORT = '8095';
```

Comment:
I would also expose a `RIF_RELAY_URL` constant to be used when we need both

Reply:
Could we read these parameters from the config file?

Reply:
Good catch.